onnxruntime/contrib_ops/rocm/bert/gemm_fast_gelu_tunable.cuh

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <hip/hip_runtime.h>
+#include <memory>
+
+#include "contrib_ops/rocm/bert/fast_gelu_impl.h"
+#include "contrib_ops/rocm/bert/gemm_fast_gelu_ck.cuh"
+#include "contrib_ops/rocm/bert/gemm_fast_gelu_common.h"
+#include "core/providers/rocm/tunable/gemm.h"
+#include "core/providers/rocm/tunable/rocm_tunable.h"
+
+namespace onnxruntime {
+namespace contrib{
+namespace rocm {
+namespace blas {
+namespace internal {
+
+template <typename T>
+Status GemmFastGeluUnfused(const GemmFastGeluParams<T>* params) {
+  namespace column_major = onnxruntime::rocm::tunable::blas::column_major;
+  if (column_major::Gemm(params->tuning, params->stream, params->handle,
+                         params->opb, params->opa,
+                         params->n, params->m, params->k,
+                         params->alpha, params->b, params->ldb, params->a, params->lda,
+                         params->beta, params->c, params->ldc) != Status::OK()) {
+    return Status(common::ONNXRUNTIME, common::FAIL, "GemmFastGelu call column_major::Gemm failed");
+  }
```

Comment:
You should not re-interpret the returned `Status` from another function. Just use `ORT_RETURN_IF_ERROR` is sufficiet here.

Reply:
Done.